mapago-client.go

```diff
@@ -7,79 +7,78 @@ import "os"
 import "sync"
 import "time"
 import "strconv"
+import "strings"
 import "github.com/protocollabs/mapago/control-plane/ctrl/client-protocols"
 import "github.com/protocollabs/mapago/measurement-plane/tcp-throughput"
 import "github.com/protocollabs/mapago/measurement-plane/udp-throughput"
+import "github.com/protocollabs/mapago/measurement-plane/quic-throughput"
 import "github.com/protocollabs/mapago/control-plane/ctrl/shared"
 
 var CTRL_PORT = 64321
 var DEF_BUFFER_SIZE = 8096 * 8
 var CONFIG_FILE = "conf.json"
-var MSMT_INFO_INTERVAL = 2
-var MSMT_STOP_INTERVAL = 10
 var MSMT_STREAMS = 1
+
 var msmtIdStorage map[string]string
 var idStorage map[string]string
 var msmtStorageInited = false
 var idStorageInited = false
-var streamCount int 
-var msmtListenAddr string
-var msmtCallSize int
 var seqNo uint64
+var streams *int 
+var serverAddr *string
+var bufLength *int
+var msmtUpdateTime *uint
+var msmtTime *uint
 
 func main() {
 	ctrlProto := flag.String("ctrl-protocol", "tcp", "tcp, udp or udp_mcast")
 	ctrlAddr := flag.String("ctrl-addr", "127.0.0.1", "localhost or userdefined addr")
 	port := flag.Int("ctrl-port", CTRL_PORT, "port for interacting with control channel")
 	callSize := flag.Int("call-size", DEF_BUFFER_SIZE, "control application buffer in bytes")
-	msmtType := flag.String("msmt-type", "tcp-throughput", "tcp-throughput or udp-throughput")
-	msmtStreams := flag.Int("msmt-streams", MSMT_STREAMS, "setting number of streams")
-	msmtLAddr := flag.String("msmt-listen-addr", "127.0.0.1", "localhost or userdefined addr")
-	msmtCSize := flag.Int("msmt-call-size", DEF_BUFFER_SIZE, "msmt application buffer in bytes")
-	
+	module := flag.String("module", "tcp-throughput", "tcp-throughput or udp-throughput")
+	streams = flag.Int("streams", MSMT_STREAMS, "setting number of streams")
+	serverAddr = flag.String("addr", "127.0.0.1", "localhost or userdefined addr")
+	bufLength = flag.Int("buffer-length", DEF_BUFFER_SIZE, "msmt application buffer in bytes")
+	msmtUpdateTime = flag.Uint("update-interval", 2, "msmt update interval in seconds")
+	msmtTime = flag.Uint("msmt-time", 10, "complete msmt time in seconds")
```

Comment:
default 10? Ich meine ist das der default wie lange übertragen wird? Wenn ich mapago --help aufrufe, wird das dann angezeigt? Ich glaube das fehlt noch bei der ausgaben, ich muss erst in den quellcode gehen um die defaults mit anzuzeigen. Der string sollte das default printen